src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/facebook/FacebookSettingsPageObject.java

```diff
@@ -64,14 +66,19 @@ public void removeAppIfPresent() {
           }
         }
       }
+      PageObjectLogging.log("removeApp", "Wikia App removed", true);
     }
-    PageObjectLogging.log("removeApp", "Wikia App removed", true);
   }
 
   /**
    * This method verifies if App is present on facebook apps list. It searches for Wikia string.
    */
   private boolean isAppPresent() {
+    try {
+      Thread.sleep(3000);
```

Comment:
what is the reason for using arbitrary sleep, is any wait method not aplicable here ?